src/Hero/Hero.css

```diff
@@ -1,3 +1,10 @@
 .hero {
-  border: solid 2px red;
+  background: url('https://images.unsplash.com/photo-1578632749014-ca77efd052eb?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1170&q=80') center;
```

Comment:
Could maybe see if adding 'no-repeat' after the url here instead of having a separate line of code 'background-repeat: no-repeat'. If that doesn't work, then this is just fine!